src/components/ciPipeline/ciPipeline.service.ts

```diff
@@ -297,20 +299,21 @@ function createMaterialList(ciPipeline, gitMaterials: MaterialType[], gitHost: G
     const ciMaterialSet = new Set()
 
     if (ciPipeline) {
-        materials = ciPipeline.ciMaterial?.map((mat) => {
-            ciMaterialSet.add(mat.gitMaterialId)
-            return {
-                id: mat.id,
-                gitMaterialId: mat.gitMaterialId,
-                name: mat.gitMaterialName,
-                type: mat.source.type,
-                value: mat.source.value,
-                isSelected: true,
-                gitHostId: gitHost ? gitHost.id : 0,
-                regex: mat.source.regex,
-                isRegex: mat.isRegex,
-            }
-        }) || []
+        materials =
+            ciPipeline.ciMaterial?.map((mat) => {
```

Comment:
mat type?

Reply:
As discussed, not making any changes around ci material (mat) types as it's used at multiple places throughout CI/CD flow.

We can cover this in a different enhancement task.